### ISSUE TITLE
Gale-Proof Cloak missing from Ascension Vendor and even from the Arsenal

### ISSUE THREAD

**Issue by Barogor** [gamefreedomgit/Frostmourne#2052]
The Gale-Proof Cloak is missing from the Ascension Vendor and even from the Arsenal.
While in both other Obsidian Sanctum Item's are in, even from 25 3D, also EoE item's are obtainable. 
It's probably even missing from Asc-Bosses loottable.

https://www.wowhead.com/wotlk/item=43988/gale-proof-cloak

**Comment by oneadanut** [comment]
It's a 200 ilvl item dropping in OS10. Ascended vendor and Arsenal sell only 213/226 items that are found in 25 man raids.

**Comment by Barogor** [comment]
That's not right. As you can see in the Following Picture. There are 200 Item Level Item's.

![Unbenannt](https://github.com/user-attachments/assets/b80f18a4-9fdb-4765-96aa-4c9dbfc43997)

**Comment by RadeghostWM** [comment]
Should be added to EoA vendor (ascended 1)
